### ISSUE TITLE
Clarity Tabs crash with Angular 9

### ISSUE THREAD

**Issue by TekSiDoT** [vmware/clarity#4171]
## Describe the bug

Clarity Tabs crash at runtime with the following error:

```
zone-evergreen.js:365 ERROR Error: ASSERTION ERROR: index
    at throwError (core.js:1364)
    at assertLessThan (core.js:1338)
    at ViewContainerRef_._adjustIndex (core.js:17412)
    at ViewContainerRef_.insert (core.js:17345)
    at ViewContainerRef_.createComponent (core.js:17307)
    at new ClrTabLink (clr-angular.js:17440)
    ...
```

## How to reproduce

I have created a basic reproduction example:
* new Angular project
* Update Angular (9.0.0-rc.7)
* Install Clarity (3.0.0-rc.5)
* Serve -> Open Page -> Open Dev Tools

Published on GH: https://github.com/TekSiDoT/ng9-clr-tabs-repro
Stackblitz does not yet seem to properly support either Angular 9 or its CLI.

## Expected behavior

Tabs should be usable without runtime crash.

## Versions

**App**

* Angular: 9.0.0-rc.7
* Node: -
* Clarity: 3.0.0-rc.5

**Device:**

* Type: MB Pro 2016
* OS: Catalina
* Browser: Checked with Chrome / FF latest

## Additional notes

Related with https://github.com/angular/angular/issues/33924 . However, the merged Fix (https://github.com/angular/angular/pull/34052) (published in Angular 9.0.0-rc.4) does not seem to have fixed the issue for Clarity.

**Comment by gnomeontherun** [comment]
Thanks, we're still slogging through the updates and we'll add this to our list of things to review.

**Comment by gnomeontherun** [comment]
@TekSiDoT have you tried this with latest Clarity and Angular rc?

**Comment by TekSiDoT** [comment]
I had to work around https://github.com/vmware/clarity/issues/4185, unfortunately, the issue persists.

**Comment by spitfjre** [comment]
This error also occurs in Clarity `2.3.1` in combination with Angular `9.0.0-rc.9`.

**Comment by TekSiDoT** [comment]
Tried again with Clarity@3.0.0-next.7 and Angular@9-rc.11, error is still reproducable, but the error message changed a bit:

![image](https://user-images.githubusercontent.com/10874046/73243207-546d1400-41a7-11ea-9004-c05ec916d990.png)

**Comment by gnomeontherun** [comment]
We haven't released a version of Clarity that properly supports Angular 9, see #4232. This should be resolved in that branch, but leaving open for the moment to ensure.

**Comment by ImranAhmed** [comment]
Angular 9 has now been released. Seeing this issue with Angular 9.0.0 and Clarity 2.3.2.

**Comment by leonelvsc** [comment]
Angular 9 released and the same happens with clarity@3.0.0-next.7

**Comment by hippee-lee** [comment]
We are still addressing issues with angular 9. I should have a release in the next few days that addresses the issues.

**Comment by gnomeontherun** [comment]
Please follow #4232 PR for updates and wait for our next release.

**Comment by gnomeontherun** [comment]
Fixed by #4232

**Comment by TekSiDoT** [comment]
Confirmed Working, thank you!
